charts/apisix/values.yaml

```diff
@@ -168,6 +168,42 @@ serviceAccount:
 rbac:
   create: false
 
+deployment:
+  # -- Apache APISIX deployment mode
+  # Optional: traditional, decoupled
+  #
+  # ref: https://apisix.apache.org/docs/apisix/deployment-modes/
+  mode: traditional
+
+  # -- Deployment role
+  # Optional: traditional, data_plane, control_plane
+  #
+  # ref: https://apisix.apache.org/docs/apisix/deployment-modes/
+  role: "traditional"
+
+  # -- used for control_plane deployment mode
+  controlPlane:
+    # -- conf Server address
+    confServerListen: "0.0.0.0:9280"
+    # -- secret name used by conf Server
+    certsSecret: ""
+    # -- conf Server CA cert name
+    cert: ""
```

Comment:
What's the correlation of these three filenames and the keys in the secret object?

Reply:
Their names are the names of the files in the secret

Reply:
It should be written to the README, or people don't know it.